api/server/cluster.go

```diff
@@ -135,6 +135,50 @@ func (c *clusterApi) inspect(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (c *clusterApi) deactivateClusterDomain(w http.ResponseWriter, r *http.Request) {
```

Comment:
remove these since we have the sdk calls

Reply:
got accidentally added to this PR.